src/unix/notbsd/android/b64.rs

```diff
@@ -10,3 +11,7 @@ s! {
 }
 
 pub const SYS_gettid: ::c_long = 178;
+
+extern {
+    pub fn timegm(tm: *const ::tm) -> time64_t;
```

Comment:
You should actually be able to change these references to `::time64_t` and avoid the duplicate definition of `time64_t` above I believe